### ISSUE TITLE
Expense : begin/end date : client validation

### ISSUE THREAD

**Issue by pgayvallet** [publicissapient-france/xndf#36]
Actually, users can select an endDate older than given startDate. 

There should be a clientside validation to block this behaviour.

**Comment by jeantil** [comment]
Agreed, even better would be to automatically re-default the end date based on the selected start date ...
